niworkflows/utils/spaces.py

```diff
@@ -148,6 +148,8 @@ class Reference:
     """The dictionary of specs."""
     standard = attr.ib(default=False, repr=False, type=bool)
     """Whether this space is standard or not."""
+    cifti = attr.ib(default=False, repr=False, type=bool)
+    """Whether this space is a CIFTI space or not."""
```

Comment:
Alternatively, I could use a new `dim` value, like 2.5, instead of adding this attribute.